py3status/modules/imap.py

```diff
@@ -52,23 +63,16 @@ def check_mail(self):
 
         response = {'cached_until': self.py3.time_in(self.cache_timeout)}
 
-        if not self.new_mail_color:
-            self.new_mail_color = self.py3.COLOR_GOOD
-
-        if mail_count == 'N/A':
-            response['full_text'] = mail_count
-        elif mail_count != 0:
-            response['color'] = self.new_mail_color
-            response['full_text'] = self.py3.safe_format(
-                self.format, {'unseen': mail_count}
-            )
+        if mail_count == self.msg_unavailable:
```

Comment:
this is broken unless `self.msg_unavailable == 'N/A'` it would be better if ` _get_mail_count()` returned `None` and we test for that